<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ip_config_dialogClass</class>
 <widget class="QDialog" name="ip_config_dialogClass">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改IP地址</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>340</width>
     <height>230</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioButton_dhcp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>自动分配IP(DHCP)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_lla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>自动分配IP(LLA)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_static">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>静态 IP</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupbox_ip_config">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>271</width>
      <height>110</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_ip_address">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>IP 地址:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subnet_mask">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>子网掩码:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_default_gate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>默认网关:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
